bin/olly.ml

```diff
@@ -191,9 +193,11 @@ let trace format trace_filename exec_args =
           ~name:(Runtime_events.runtime_phase_name phase)
           ~time:(ts |> ts_to_int |> int_to_span)
       in
+      let extra = Olly_custom_events.v trace_file doms in
       let init () = () in
       let cleanup () = Trace.close trace_file in
-      olly ~runtime_begin ~runtime_end ~init ~lifecycle ~cleanup exec_args
+      olly ~extra ~runtime_begin ~runtime_end ~init ~lifecycle ~cleanup
```

Comment:
Did not do a full review. Do we want to add custom events to json as well?

Reply:
For completeness sake, we should.